### DISCUSSION TITLE
How can i make an image button?

### DISCUSSION THREAD

**Discussion by zolotaya-ruda** [hecrj/iced#1985] [Q&A]
I'm trying make it like [this](https://github.com/iced-rs/iced/issues/692)

but it seems to me that this is an outdated option, is it so? and then what are the ways to make such a button?

**Comment by Q4REAL** [comment]
I believe a `button` can contain most other widgets. This seemed to work fine for me:
```
button(
    image("src/image.png").height(32).width(32)
).into()
```
just make sure to have the `image` feature enabled in your `cargo.toml`
